src/patternfly/components/Button/button-icon.hbs

```diff
@@ -2,5 +2,9 @@
   {{#if button-icon--attribute}}
     {{{button-icon--attribute}}}
   {{/if}}>
-  {{> @partial-block}}
+  {{#if button-icon--name}}
```

Comment:
i think this should be in its own issue/pr since its just affecting the button and we can keep track of it